torch/ao/quantization/fx/convert.py

```diff
@@ -485,7 +485,8 @@ def convert(
         is_standalone_module: bool = False,
         _remove_qconfig_flag: bool = True,
         qconfig_mapping: Union[QConfigMapping, Dict[str, Any], None] = None,
-        backend_config: Union[BackendConfig, Dict[str, Any], None] = None) -> torch.nn.Module:
+        backend_config: Union[BackendConfig, Dict[str, Any], None] = None,
+        is_decomposed_qtensor: bool = False) -> torch.nn.Module:
```

Comment:
Do we want to make this a private API (i.e. `_is_decomposed_qtensor`)? Otherwise I feel we should mention it in the docstring

Reply:
this is not a public facing convert, I feel it should be fine, but I can add some docs for it as well